src/Workspaces/Remote/Core/RemoteWorkspacesResources.resx

```diff
@@ -213,7 +213,7 @@
   <data name="FeatureName_StackTraceExplorer" xml:space="preserve">
     <value>Stack Trace Explorer</value>
   </data>
-  <data name="FeatureName_SolutionEventsAggregation" xml:space="preserve">
-    <value>Solution Events</value>
+  <data name="FeatureName_LegacySolutionEventsAggregation" xml:space="preserve">
+    <value>Legacy Solution Events</value>
```

Comment:
>Legacy [](http://example.com/codeflow?start=11&length=6)

I wouldn't include "legacy" in the display name. Don't think we need to advertise obsoleteness to customers, if they hit gold bar caused by this service.

Reply:
sgtm.